### DISCUSSION TITLE
Dịch prefetch

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#141] [Terminology] [answered]
Context:
> By calling `prefetch(1)` at the end, we are creating a dataset that will do its best to always be one batch ahead.
In other words, while our training algorithm is working on one batch, the dataset will already be working in parallel on getting the next batch ready (e.g., reading the data from disk and preprocessing it). This can improve performance dramatically...

Từ này nên dịch là gì nhỉ?
* tiền nạp
* nạp trước
* nạp sớm

cc @mlbvn/handson-ml

**Comment by ngcthuong** [comment]
Em cho thêm bối cảnh từ này vào nhé

**Reply by minhduc0711** [reply]
Em edit post gốc rồi ạ :+1:

**Comment by cuongvng** [comment]
1 vote cho "nạp sớm"

**Reply by thanhcsf** [reply]
+1 cho nạp sớm.

**Comment by ngcthuong** [comment]
Nạp đệm?

**Comment by minhduc0711** [comment, marked as answer]
"nạp sớm" hiện đang có nhiều vote hơn nên mình tạm chốt như vậy đi ạ
